admin_guide/managing_projects.adoc

```diff
@@ -0,0 +1,171 @@
+
+OpenShift developers can set a node selector on their project configuration if they wish to restrict nodes even further. This will be added to the cluster-wide node selector, meaning it will be more restrictive.
+
+.Creating a Cluster-wide Node Selector
+
+To create a project with a node selector, use the `--node-selector` option. The following creates a new project named `myproject` and dictates that pods be deployed onto nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project myproject \
+    --node-selector='type=user-node,region=east'
+----
+====
+
+Once this command is run, this becomes the adminstrator-set node selector for all pods contained in the indicated project.
+
+[NOTE]
+====
+While the `new-project` subcommand is available for both `oadm` and `oc`, the cluster administrator and developer commands respectively, creating a new project with a node selector is only available with the `oadm` command. The `new-project` subcommand is not available to project developers when self-provisioning projects.
+====
+
+Using the above command adds an `annotation` section to the project configuration. Changing the `openshift.io/node-selector` value overrides the default:
+
+====
+----
+...
+metadata:
+  annotations:
+    openshift.io/node-selector: type=user-node,region=east
+...
+----
+====
+
+If `openshift.io/node-selector` is set to an empty string (`oadm new-project --node-selector=""`), the project will not have an adminstrator-set node selector, even if the cluster-wide default has been set. This means that, as an OpenShift administrator, you can set a default to restrict developer projects to a subset of nodes and still enable infrastructure or other projects to schedule the entire cluster.
+
+However, you can still dictate node selector values for all projects with a node selector value. If a project has been created with the above annotation (`openshift.io/node-selector: type=user-node,region=east`) and a developer sets another node selector on a pod in that project, for example `clearance=classified`, the pod will only ever be scheduled on nodes that have all three labels (`type=user-node`, `region=east` and `clearance=classified`). If they decided they wanted to schedule on `region=west` and set a pod node selector accordingly, they would be demanding nodes with labels `region=east` and `region=west`, which cannot work because labels can only be set to one value, so it would never be scheduled. Thus the developer can, per-pod, augment (but not override) the project node selector.
+
+Alternatively, you can edit the *_/etc/origin/master/master-config.yaml_* master configuration file and add a node selector value. This applies the node selector to pods created in all projects without a specified `*nodeSelector*` value:
```

Comment:
This would make more sense to me at the top of the section. First discuss the global default, then the project-specific override, then the pod-specific narrowing of node selector.